ikasaneip/dashboard/war/src/main/java/org/ikasan/dashboard/ui/topology/component/BusinessStreamTab.java

```diff
@@ -427,10 +441,199 @@ public void buttonClick(ClickEvent event)
+        final Node node3 = new Node(3,"Node 3");
+        
+//        node3.setImage("https://vaadin.com/download/vaadin-icons/png/archive.png");
+        Node node4 = new Node(4,"Node 4");
+        
+//        node4.setImage("https://vaadin.com/download/vaadin-icons/png/archive.png");
+        Node node5 = new Node(5,"Node 5");
+        
+//        node5.setImage("https://vaadin.com/download/vaadin-icons/png/archive.png");
+        Node node6 = new Node(6,"Node 6");
+       
+//        node6.setImage("https://vaadin.com/download/vaadin-icons/png/archive.png");
+        
+        final Node murex = new Node(10,"MUREX");
+        murex.setShape(Node.Shape.square);
+        Color mc = new Color();
+        mc.setBackgroundColor("#504faf");
+        mc.setColor("#504faf");
+        mc.setHoverColor("#504faf");
+        mc.setHighlightColor("#504faf");
+        murex.setColor(mc);
+
+        //create edges
+        final Edge edge1 = new Edge(node2.getId(),node1.getId(), 2);
+        edge1.setStyle(Edge.Style.arrow);
+        
+        Edge edge2 = new Edge(node3.getId(),node1.getId(), 2);
+        edge2.setStyle(Edge.Style.arrow);
+        Edge edge3 = new Edge(node1.getId(),node5.getId(), 2);
+        edge3.setStyle(Edge.Style.arrow);
+        Edge edge4 = new Edge(node1.getId(),node4.getId(), 2);
+        edge4.setStyle(Edge.Style.arrow);
+        Edge edge5 = new Edge(node1.getId(),node6.getId(), 2);
+        edge5.setStyle(Edge.Style.arrow);
+        
+        Edge edge6 = new Edge(murex.getId(),node6.getId(), 2);
+        edge6.setStyle(Edge.Style.arrow);
+
+        networkDiagram.addNode(node1);
+        networkDiagram.addNode(node2,node3,node4,node5,node6, murex);
+        networkDiagram.addEdge(edge1,edge2,edge3,edge4,edge5,edge6);
+        
+        DragAndDropWrapper layoutWrapper =
+                new DragAndDropWrapper(networkDiagram);
+        
+
+        // Handle moving components within the AbsoluteLayout
+        layoutWrapper.setDropHandler(new DropHandler() 
+        {
+            public AcceptCriterion getAcceptCriterion() 
+            {
+                return AcceptAll.get();
+            }
+
+            public void drop(DragAndDropEvent event) 
+            {            	
+            	final DataBoundTransferable t = (DataBoundTransferable) event
+                        .getTransferable();
+		
+				if(t.getItemId() instanceof Module)
+				{
+					final Module module = (Module) t
+							.getItemId();
+					
+					Node node = new Node(module.getId().intValue(), module.getName());
+					
+					node.setImage("https://vaadin.com/download/vaadin-icons/png/archive.png");
+					
+					Edge edge = new Edge(node.getId(),node1.getId(), 2);
+			        edge.setStyle(Edge.Style.arrow);
+					
+					networkDiagram.addNode(node);
+					networkDiagram.addEdge(edge);					
+	            }
+            }
+            
+        });
+        
+
+        ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
```

Comment:
Thanks for reminding me of this. This tab is a work in progress which will not be displayed in production. I have removed the executor for now.